### PR TITLE
Pg json support

### PR DESCRIPTION
**Pull request by willyborankin**
PgSQL JSON/JSONB support for predefined tables

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Aiven Oy
+ * Copyright 2020 Aiven Oy
```
Comment (HelenMel) [inline build.gradle:2]: I'm not sure, if we should update a date. Here is a discussion, https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated#:~:text=Technically%2C%20you%20should%20update%20a,just%20to%20update%20the%20year. Most popular answer is that we shouldn't

File: src/integration-test/java/io/aiven/connect/jdbc/SchemaRegistryContainer.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * limitations under the License.
+ */
+
+package io.aiven.connect.jdbc;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.Base58;
+
+public final class SchemaRegistryContainer extends GenericContainer<SchemaRegistryContainer> {
```
Comment (HelenMel) [inline src/integration-test/java/io/aiven/connect/jdbc/SchemaRegistryContainer.java:23]: This class looks familiar. Maybe move it to commons?

File: src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    private final ArrayDeque<String> createdConnectors = new ArrayDeque<>();
+
+    private final Random partitionRnd = new Random();
+
+    @BeforeEach
+    void createDbTable() throws Exception {
+        try (final var stm = pgConnection.createStatement()) {
+            final var createTableSql =
+                    String.format(
+                            "CREATE TABLE %s ("
```
Comment (HelenMel) [inline src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java:66]: Stupid question. Should we also add timestamp as one of the fields?? Maybe also offset

File: build.gradle (hunk 1 above)
Comment (willyborankin) [reply build.gradle:2]: Well before that we upgraded. Let's keep it so far and discuss this question in the team ok?

File: src/integration-test/java/io/aiven/connect/jdbc/SchemaRegistryContainer.java (hunk 2 above)
Comment (willyborankin) [reply src/integration-test/java/io/aiven/connect/jdbc/SchemaRegistryContainer.java:23]: True, we need to put it in common ... There are a lot of open questions how to it better ...

File: src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java (hunk 3 above)
Comment (willyborankin) [reply src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java:66]: Nope. The purpose is to show how to use JSON/JSONB in DB for existing tables.

**Comment by ivanyu** [comment]
Re: everything about Commons. Please note that this connector is not using Commons at the moment. Commons is largely tailored for object storage connectors, adding it as a dependency for this connector for the sake of a test class isn't maybe the best idea.
(Actually, we need to sit and think about this more strategically, what would be the ideal dev/distribution setup for all our connectors.)

**Comment by willyborankin** [comment]
> Re: everything about Commons. Please note that this connector is not using Commons at the moment. Commons is largely tailored for object storage connectors, adding it as a dependency for this connector for the sake of a test class isn't maybe the best idea.
> (Actually, we need to sit and think about this more strategically, what would be the ideal dev/distribution setup for all our connectors.)

True. We had a quick video chat about @HelenMel's quistions

**Comment by HelenMel** [comment]
@ivanyu we discussed with Andrei technical details. Everything looks OK. The only thing I'm not sure about is Copyrights. But we could discuss it with a team after

**Review by HelenMel — APPROVED** [review]

**Pull request merged**